.github/workflows/doc-build.yml

```diff
@@ -227,7 +227,7 @@ jobs:
         shell: bash
         run: |
           export PYTHONFAULTHANDLER=1
-          xvfb-run make -C doc ${{ inputs.builder }} SPHINXOPTS="-j auto -W --keep-going"
+          xvfb-run make -C doc ${{ inputs.builder }} SPHINXOPTS="-j auto -W --keep-going" | tee doc_build.log
```

Comment:
Do you want to keep the ``tee`` command or was it only for debugging?

Reply:
Let's keep it and attach the output to the artifacts :)